### PR TITLE
Adding protection to whitelist add and remove.

### PR DESCRIPTION
**Pull request by 75efb6**
Adding protection via using a key to protect those endpoints, the key needs to be in .env

Usage: {endpoint}?key={your key}&bid={mapid}

### CODE REVIEW

**Comment by 75efb6** [comment]
Wait, usage got f'ed

**Comment by 75efb6** [comment]
The var on .env is: WL_KEY

File: handlers/api.py [hunk 1]
```diff
@@ -205,6 +210,8 @@ async def whitelist_add():
 @bp.route('/wl_remove', methods=['GET'])
 async def whitelist_remove():
     data = request.args
+    if key != os.getenv("WL_KEY"):
```
Comment (unclem2) [inline handlers/api.py:214]: Should be `data.get('key')`, ill merge and change that

**Review by unclem2 — COMMENTED** [review]
Ill merge and update it. Thanks for contributing

**Pull request merged**